packages/jaeger-ui/src/components/DeepDependencies/Graph/DdgNodeContent/calc-positioning.tsx

```diff
@@ -102,7 +102,7 @@ export function _initOpSpan() {
  *     recursive call stack.
  * @returns {number} - Width of narrowest rectangle that contains given words across given lines.
  */
-function calcWidth(lengths: number[], lines: number, longestThusFar: number = 0): number {
+function calcWidth(lengths: number[], lines: number, longestThusFar = 0): number {
```

Comment:
what is the reason for this change? Is a new linter rule complaining about it? It seems like a normal syntax.

Reply:
By default it's giving error, called no-inferrable-types. It disallow explicit type declarations for variables or parameters initialized to a number, string, or boolean.